bin/list

```diff
@@ -161,6 +161,34 @@ const renderDirectory = async dir => {
   return render(details)
 }
 
+
+
+// To enhance, one can take inspiration from https://github.com/jfhbrook/node-ecstatic/blob/fe17a5623c18319f82101b6549e9e2e526e03d1f/lib/ecstatic.js#L284
+const isNotModified = (req, etag, fileStats) => {
+  try {
+    // Etag comparison
+    const requestEtag = req.headers['if-none-match'];
+    if (requestEtag && (requestEtag === etag)) {
+      return true;
+    }
+    // Last-modified comparison
+    // TODO: Original code, left untouched, but for me it's not working at all
+    // The algo seems wrong to be and we don't even set a Last-modified header in response
+    let requestDate = req.headers['if-modified-since']
```

Comment:
Was the code working before your changes? If it wasn't and still isn't working, what's the value of keeping it around (based on your comment)?